.env.sample

```diff
@@ -12,5 +12,6 @@ WARNINGS_THRESH=x
 PAPERTRAIL_HOST=x
 PAPERTRAIL_PORT=x
 TWITTER_BEARER_TOKEN=x
+GUILD_ID=852656268090802263=x
```

Comment:
i prefer that we automatically get the guild from the guild service on startup, de-register all commands, and then re-register whichever ones still exist.